src/components/CourseTable/index.jsx

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
-import apiClient from '../../data/apiClient';
 
-function CourseTable() {
+import TableContainer from '../../containers/TableContainer';
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
+
+const CourseTable = () => {
+  const courseTableColumns = [
+    {
+      label: 'Course Name',
+      key: 'title',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Number',
+      key: 'key',
+      columnSortable: true,
+    },
+    {
+      label: 'Owner',
+      key: 'owners',
+      columnSortable: false,
+    },
+    {
+      label: 'Modified',
+      key: 'modified',
+      columnSortable: true,
+    },
+  ];
+
+  const formatCourseData = courses => courses.map(course => ({
+    ...course,
+    owners: course.owners ? course.owners.map(owners => owners.name).join(', ') : '',
```

Comment:
In my local data I am not seeing owner coming over with my test courses.  I need to check where it is set.  This might just be something we are not currently setting with the new Create API.

Reply:
Looks like an issue with the new Course creation code.  I will move it to separate ticket.